pages/client_page.vue

```diff
@@ -0,0 +1,240 @@
+        </div>
+        <div class="client_card">
+            <div class="card_header">Karta rachunkowa</div>
+            <div class="card_number">**** **** **** 2500</div>
+            <div class="label_card_number">Numer karty</div>
+            <div class="card_valid_date">07 / 24</div>
+            <div class="label_card_valid_date">Data ważności</div>
+            <div class="card_type">Mastercard</div>
+            <div class="label_card_type">Rodzaj karty</div>
+            <button class="update-card">Aktualizuj dane karty</button>
+        </div>
+    </div>
+</template>
+
+<script setup>
+</script>
+
+<style lang="scss">
+.container {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 900px 400px;
+    grid-template-rows: 2fr 5fr 1fr;
+    column-gap: 20px;
+    row-gap: 10px;
+    margin: 0 60px;
+}
+
+.container > div, button {
+    background-color: $background400;
+    padding: 20px;
+    border-radius: 10px;
+}
+
+button {
+    grid-column: 1;
+    grid-row: 1;
+    justify-self: start;
+    font-size: 18px;
+    color: $text500;
+    align-self: end;
+    padding: 10px 20px;
+}
+
+div.client_profile {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $background500;
+    grid-row: 1;
+    grid-column: 1 / span 2;
+    justify-self: end;
+    align-self: end;
+}
+
+.client_profile > div {
+    margin-right: 20px;
+    font-size: 30px;
+}
+
+.client_profile > img {
+    width: 100px;
+    height: 100px;
+    border-radius: 100%;
+}
+
+.client_profile > img, div {
+    float: left;
+}
+
+.client_data {
+    grid-column: 1;
+    grid-row: 2;
+    font-size: 30px;
+    padding: 20px 40px;
+    display: grid;
+    grid-template-columns: 340px 500px;
+    column-gap: 20px;
+    justify-items: stretch;
+}
+
+.data_header {
+    margin-bottom: 30px;
+}
+
+.username, .names, .email {
+    grid-column: 1;
+    font-size: 24px;
+}
+
+.label_username, .label_names, .label_email {
+    grid-column: 1;
+    font-size: 15px;
+}
+
+.subscriptions {
+    border: 2px $text500 solid;
+    border-radius: 10px;
+    grid-column: 2;
+    grid-row: 1 / span 7;
+    justify-self: stretch;
+    align-self: stretch;
+    row-gap: 10px;
+    padding: 0 0 20px 0;
+}
+
+.subscriptions_header {
+    font-size: 26px;
+    text-align: center;
+    padding: 20px 0;
+    background: $primary500;
+    border-top-right-radius: 10px;
+    border-top-left-radius: 10px;
+    width: 501px;
+    transform: translate(-2px, -2px);
+}
+
+.balance_state {
+    font-size: 20px;
+    padding: 20px 20px;
+    width: 460px;
+}
+
+.bundles_label {
+    font-size: 14px;
+    padding: 10px 20px;
+    width: 460px;
+}
+
+.bundles {
+    padding: 0 20px;
+    width: 460px;
+    margin-top: 10px;
+}
+
+.bundle {
+    background-color: #000000;
+    width: 460px;
+    height: 50px;
+    margin-bottom: 10px;
+    border-radius: 10px;
+    text-align: center;
+}
+
+.buttons {
+    grid-column: 1;
+    display: grid;
```

Comment:
proponuję używać częściej flexa, jeżeli przyciski są w tej samej osi najpewniej osiągniesz to samo za pomocą czegoś w tym stylu:
```scss
display: flex
flex-direction: row
justify-content: space-between
align-items: center
```

Reply:
![image](https://github.com/documentoviscode/splashy-frontend/assets/90265591/f2ffd77e-feb2-4bc9-acd3-fd5b2cbe0e98)
nie taki efekt chciałem uzyskać...

Reply:
pisane na oko